xbmc/GUIInfoManager.cpp

```diff
@@ -436,152 +680,109 @@ const infomap player_labels[] =  {{ "hasmedia",         PLAYER_HAS_MEDIA },
                                   { "frameadvance",     PLAYER_FRAMEADVANCE },
                                   { "icon",             PLAYER_ICON }};
 
-/// \page modules__General__List_of_gui_access
-/// @{
-/// \table_start
-///   \table_row3{   <b>`Player.Art(fanart)`</b>,
-///                  \anchor Player_Art_fanart
-///                  _string_,
-///     Fanart Image of the currently playing episode's parent TV show
-///   }
-///   \table_row3{   <b>`Player.Art(thumb)`</b>,
-///                  \anchor Player_Art_thumb
-///                  _string_,
-///     Returns the thumbnail image of the currently playing item.
-///   }
-///   \table_row3{   <b>`Player.Art(poster)`</b>,
-///                  \anchor Player_Art_poster
+/// \page modules__infolabels_boolean_conditions
+///   \table_row3{   <b>`Player.Art(type)`</b>,
+///                  \anchor Player_Art_type
 ///                  _string_,
-///     Returns the poster of the currently playing movie.
+///     @return The Image for the defined art type for the current playing ListItem.
+///     @param type - The art type. The type is defined by scripts and scrappers and can have any value.
+///     Common example values for type are:
```

Comment:
I would say:

The art type. Common example values for type are:
[ - list-]
However type can be any value depending on what is provided by scripts and scrapers, or manually added by the user.